core/src/main/resources/schema/app/20210719141901_CB-13065_remove_datalakeresource_and_servicedescriptor.sql

```diff
@@ -0,0 +1,69 @@
+-- // CB-13065 removing datalakeresources and servicedescriptor
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE stack DROP COLUMN datalakeresourceid;
+
+DROP TABLE datalakeresources_rdsconfig;
+
+DROP TABLE servicedescriptor;
+
+DROP TABLE datalakeresources;
+
+-- //@UNDO
+-- SQL to undo the change goes here.
+
+CREATE TABLE IF NOT EXISTS datalakeresources (
+    id bigserial NOT NULL,
+    datalakestack_id bigserial,
+    name character varying(255) NOT NULL,
+    datalakeambariip character varying(255) NOT NULL,
+    datalakeambarifqdn character varying(255) NOT NULL,
+    workspace_id bigint NOT NULL,
+    datalakecomponents TEXT NOT NULL,
+    datalakeclustermanagerip varchar(255),
+    datalakeclustermanagerfqdn varchar(255),
+    datalakeclustermanagerurl varchar(255),
+    environmentcrn varchar(255),
+    datalakeambariurl VARCHAR(255),
+    kerberosconfig_id BIGINT
+);
+ALTER TABLE ONLY datalakeresources ADD CONSTRAINT datalakeresources_pkey PRIMARY KEY (id);
+ALTER TABLE ONLY datalakeresources ADD CONSTRAINT fk_stack_datalakestack_id FOREIGN KEY (datalakestack_id) REFERENCES stack(id);
+ALTER TABLE ONLY datalakeresources ADD CONSTRAINT uk_datalakeresources_workspace_name UNIQUE (workspace_id, name);
+CREATE INDEX IF NOT EXISTS idx_datalakeresources_workspace_id_name ON datalakeresources (workspace_id, name);
+CREATE INDEX IF NOT EXISTS idx_datalakeresources_datalakestack_id ON datalakeresources (datalakestack_id);
+
+CREATE TABLE IF NOT EXISTS servicedescriptor (
+    id bigserial NOT NULL,
+    datalakeresources_id bigint NOT NULL,
+    workspace_id bigint NOT NULL,
+    servicename character varying(255) NOT NULL,
+    blueprintparams TEXT NOT NULL,
+    blueprintsecretparams TEXT,
+    componentshosts TEXT NOT NULL
+);
+ALTER TABLE ONLY servicedescriptor ADD CONSTRAINT servicedescriptor_pkey PRIMARY KEY (id);
+ALTER TABLE ONLY servicedescriptor ADD CONSTRAINT fk_servicedescriptor_datalakeresources_id FOREIGN KEY (datalakeresources_id) REFERENCES datalakeresources(id);
+CREATE INDEX IF NOT EXISTS idx_servicedescriptor_datalakeresources_id ON servicedescriptor (datalakeresources_id);
+
+DROP SEQUENCE IF EXISTS datalakeresources_datalakestack_id_seq CASCADE;
+ALTER TABLE datalakeresources ALTER COLUMN datalakestack_id DROP NOT NULL;
```

Comment:
are you sure this is part of the undo script?

Reply:
these were part of the original scripts which created and manipulated the datalakeresource table. from 2018 till recent time

Reply:
I would rather script out the CREATE TABLE scripts and paste those. At least please try out if the undo script produces the expected result, e.g. the tables are restored to the same state.